packages/web-runtime/src/container/sse.ts

```diff
@@ -62,18 +63,11 @@ export const onSSEItemRenamedEvent = async ({
 }) => {
   try {
     const sseData = fileReadyEventSchema.parse(JSON.parse(msg.data))
-
     const currentFolder = resourcesStore.currentFolder
     const resourceIsCurrentFolder = currentFolder.id === sseData.itemid
-
-    if (!resourceIsCurrentFolder && !itemInCurrentFolder({ resourcesStore, sseData })) {
```

Comment:
I ommited this check, since we already have a resource check and if its not in store its probably not in currentFolder